### PR TITLE
- Fixed some issues with PWA

### PR DESCRIPTION
**Pull request by CodeWithTheDoctor**
- "Install app" option wasn't appearing instantly because maskable.png route was incorrect. 
  - Fixed that.

### CODE REVIEW

**Comment by skyheat** [comment]
Noticed that the name of the app is SHF APP. Maybe we should change it?

File: pages/_document.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ export default function Document() {
         <meta name='theme-color' content='#671E75' />
         <link rel='manifest' href='/manifest.json' />
         <link rel='shortcut icon' href='/favicon.ico' />
```
Comment (NicholasChoong) [inline pages/_document.tsx:40]: shouldn't this be `icons/maskable.png`?

**Review by NicholasChoong — CHANGES_REQUESTED** [review]

File: pages/_document.tsx (hunk 1 above)
Comment (NicholasChoong) [reply pages/_document.tsx:40]: or maybe convert the file to `favicon.ico`?

File: pages/_document.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ export default function Document() {
         <meta name='theme-color' content='#671E75' />
         <link rel='manifest' href='/manifest.json' />
```
Comment (NicholasChoong) [inline pages/_document.tsx:39]: Web app manifest should have the filename extension 'webmanifest'.

FURTHER READING
[Learn why this is important and how to fix it on webhint.io](https://webhint.io/docs/user-guide/hints/hint-manifest-file-extension/?source=devtools)

**Review by NicholasChoong — CHANGES_REQUESTED** [review]

File: pages/_document.tsx (hunk 2 above)
Comment (CodeWithTheDoctor) [reply pages/_document.tsx:39]: Will do!

**Comment by CodeWithTheDoctor** [comment]
The name is Superhero Foods, but the short name is SHF APP... I'm not sure of a better short name. I'll change it to `SHF App` tho.
image

**Review by skyheat — APPROVED** [review]
Looks good to me!

**Pull request merged**